### ISSUE TITLE
Having a 'file' component as last component in multipage form, click submit opens file dialog

### ISSUE THREAD

**Issue by jaccovanderpol** [formio/formio.js#574]
I found that when you have a multi page form where you have the last field being a file component, when re-opening a previously edited form, the file dialog opens when I click submit, next or previous. This behaviour is because I get an <input type="file" style="opacity: 0; position: absolute;" tabindex="-1"> element just above the buttons (in html) which results the input being over/under/whatever the buttons.

Found that it has probably something to do with the buildHiddenFileInput function in src/components/file/File.js. No idea how to solve it as this element is not there on first open of the form and only after re-opening iow I don't know what it is there fore.

**Comment by jaccovanderpol** [comment]
It seems you can solve this problem by changing line in function build() in file.js:
this.inputsContainer.appendChild(this.hiddenFileInputElement);
to
this.uploadContainer.appendChild(this.hiddenFileInputElement);
Can anyone confirm this is the right solution?

**Comment by jhen1422** [comment]
This should be resolved in our 4x renderer.
Please feel free to open a new issue if the problem persists.
